### PR TITLE
Add DEM docstring

### PR DESCRIPTION
**Pull request by MLicari1**
Made a more detailed doc string for the download DEM function.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have added an explanation of what your changes do and why you'd like us to include them.
- [ ] I have written new tests for your core changes, as applicable.
- [ ] I have successfully ran tests with your changes locally.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by MLicari1** [comment]
@jlmaurer Here is the pr for the doc string we talked about at our meeting. Let me know if should make any changes to it. Thank you!

File: tools/RAiDER/dem.py [hunk 1]
```diff
@@ -57,7 +57,18 @@ def download_dem(
...
+    Download a DEM if one is not already present. 
+    Args:
+            llbounds: list/ndarry of floats   -lat/lon bounds of the area to download. Values should be ordered in the following way: [S, N, W, E]
+            writeDEM: boolean                 -write the DEM to file
+            outName: string                   -name of the DEM file   
+            buf: float                        -buffer to add to the bounds
+            overwrite: boolean                -overwrite existing DEM
+    Returns:
+            zvals: np.array         -DEM heights
+            metadata:               -metadata for the DEM
```
Comment (jlmaurer) [inline tools/RAiDER/dem.py:70]: @MLicari1 could you add some details to what is in the metadata? You can check the rasterio documentation for examples.

File: tools/RAiDER/dem.py [hunk 2]
```diff
@@ -57,7 +57,18 @@ def download_dem(
     buf=0.02,
     overwrite=False,
 ):
-    """  Download a DEM if one is not already present. """
+    """  
+    Download a DEM if one is not already present. 
+    Args:
+            llbounds: list/ndarry of floats   -lat/lon bounds of the area to download. Values should be ordered in the following way: [S, N, W, E]
+            writeDEM: boolean                 -write the DEM to file
+            outName: string                   -name of the DEM file
```
Comment (jlmaurer) [inline tools/RAiDER/dem.py:65]: Please add that if the DEM exists the default behavior is to use the existing.

**Review by jlmaurer — COMMENTED** [review]
@MLicari1 few small suggestions. Also would you be able to add a unit test that would specifically test the existing DEM behavior? You can use the DEM in `test/scenario_0/geom/` as the example. Add the test to `test/test_dem.py` and use 
```
from test import TEST_DIR
SCENARIO_DIR = os.path.join(TEST_DIR, "scenario_0")
file = os.path.join(SCENARIO_DIR, 'geom', 'warpedDEM.dem')
```
to grab the file for the test.

File: tools/RAiDER/dem.py (hunk 2 above)
Comment (MLicari1) [reply tools/RAiDER/dem.py:65]: Will do 👍. I should have some time tomorrow to work on these

**Review by jlmaurer — APPROVED** [review]

**Pull request merged**